connectors/cybereason/configurations/config.json

```diff
@@ -0,0 +1,37 @@
+{
+    "connection": {
+        "type": {
+            "type": "connectorType",
+            "displayName": "Cybereason"
+        },
+        "host": {
+            "type": "text",
+            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
+        },
+        "port": {
+            "type": "number",
+            "default": 443,
+            "min": 1,
+            "max": 65535
+        }
+    },
+    "configuration": {
+        "auth": {
+            "type": "fields",
+            "username": {
+                "type": "password"
+            },
+            "password": {
+                "type": "password"
+            }
+        },
+        "parameter": {

```

Comment:
To have this more aligned with the UDI connectors config files, I feel this should be under the connection object and instead of named "parameters" change it to "options". You would remove the outer "type" and just leave the "type": "number" under the "vulnerability_retention_period".

Reply:
Updated configuration files as suggested.